modules/mobile/app/models/mobile/v0/adapters/lighthouse_claims_overview.rb

```diff
@@ -27,8 +27,9 @@ def parse_claim(entry)
               display_title: attributes['claimType'],
               decision_letter_sent: attributes['decisionLetterSent'],
               phase: Mobile::ClaimsHelper.phase_to_number(attributes['claimPhaseDates']['phaseType']),
-              documentsNeeded: attributes['documentsNeeded'],
-              developmentLetterSent: attributes['developmentLetterSent']
+              documents_needed: attributes['documentsNeeded'],
+              development_letter_sent: attributes['developmentLetterSent'],
```

Comment:
realized that I messed up the casing when adding the other fields. this PR corrects that.